src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -112,10 +112,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
     if (isTerminalState(status)) {
       liveState.removeRunningTask(status.getTaskId());
       persistentState.removeTaskId(status.getTaskId().getValue());
-      if (reconciliationComplete()) {
-        correctCurrentPhase();
-      }
-
```

Comment:
The purpose of this call is to correct the phase when a task dies after the reconcile period is over.